tests/unit/dojson/test_dojson_hep.py

```diff
@@ -2069,48 +2069,62 @@ def test_book_link(marcxml_to_json_book):
             1409249)
 
 
-def test_acquisition_source_field():
-    """Test acquisition_source."""
+def test_acquisition_source_from__541_a_c():
     snippet = (
-        '<record>'
-        '   <datafield tag="541" ind1=" " ind2=" ">'
-        '       <subfield code="a">inspire:uid:50000</subfield>'
-        '       <subfield code="b">example@gmail.com</subfield>'
-        '       <subfield code="c">submission</subfield>'
-        '       <subfield code="d">2015-12-10</subfield>'
-        '       <subfield code="e">339830</subfield>'
-        '   </datafield>'
-        '</record>'
-    )
+        '<datafield tag="541" ind1=" " ind2=" ">'
+        '  <subfield code="a">IOP</subfield>'
+        '  <subfield code="c">batchupload</subfield>'
+        '</datafield>'
+    )  # record/1487640/export/xme
 
     expected = {
-        'source': "inspire:uid:50000",
-        'email': "example@gmail.com",
-        'method': "submission",
-        'date': "2015-12-10",
-        'submission_number': "339830",
+        'source': 'IOP',
+        'method': 'batchupload',
     }
     result = hep.do(create_record(snippet))
 
     assert expected == result['acquisition_source']
 
 
-def test_acquisition_source_field_marcxml():
-    """Test acquisition_source MARC output."""
+def test_acquisition_source_from__541_double_a_b_c_e():
+    snippet = (
+        '<datafield tag="541" ind1=" " ind2=" ">'
+        '  <subfield code="a">inspire:uid:52524</subfield>'
+        '  <subfield code="a">orcid:0000-0002-1048-661X</subfield>'
+        '  <subfield code="b">oliver.schlotterer@web.de</subfield>'
+        '  <subfield code="c">submission</subfield>'
+        '  <subfield code="e">504296</subfield>'
+        '</datafield>'
+    )  # record/1416571/export/xme
+
     expected = {
-        'a': 'inspire:uid:50000',
-        'c': 'submission',
-        'b': 'example@gmail.com',
-        'e': '339830',
-        'd': '2015-12-10'
+        'source': 'orcid:0000-0002-1048-661X',
+        'email': 'oliver.schlotterer@web.de',
+        'method': 'submission',
+        'submission_number': '504296',
     }
+    result = hep.do(create_record(snippet))
 
-    record = {"acquisition_source": {
-        'source': "inspire:uid:50000",
-        'email': "example@gmail.com",
-        'method': "submission",
-        'date': "2015-12-10",
-        'submission_number': "339830",
-    }}
+    assert expected == result['acquisition_source']
+
+
+def test_541__a_b_c_e_from_acquisition_source():
+    record = {
+        'acquisition_source': {
+            'source': 'orcid:0000-0002-1048-661X',
+            'email': 'oliver.schlotterer@web.de',
+            'method': 'submission',
+            'submission_number': '504296',
+        },
+    }
+
+    expected = {
+        'a': 'orcid:0000-0002-1048-661X',
+        'b': 'oliver.schlotterer@web.de',
+        'c': 'submission',
+        'd': None,
```

Comment:
This is because `clean_record` is not part of the `hep2marc` filters. Should it be? @kaplun